src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -177,11 +180,29 @@ const SavedSearchAlertEditForm: React.FC<SavedSearchAlertEditFormProps> = ({
                   />
                 </Flex>
 
-                <Separator my={1} />
+                <Separator my={2} />
 
                 <PriceRangeFilter />
 
                 <Separator my={2} />
+
+                <Text variant="sm-display">
+                  Tell us more about what you’re looking for
+                </Text>
+                <Spacer y={1} />
+                <TextArea
```

Comment:
**suggestion**: Looks we like we might want to extract this component so we can more easily reuse it between create and edit flows.